### ISSUE TITLE
Prototypes or wireframes for frontend

### ISSUE THREAD

**Issue by Infi-Knight** [kowainik/issue-wanted#19]
This issue is in reference for the creation of prototypes or wireframes for the frontend. It would aid in streamlining the backend as per project's requirements. Do these wireframes exist or are in the process of development? If not then would it be desirable if I start implementing them with the guidance of respected contibutors?

**Comment by ChShersh** [comment]
@Infi-Knight I agree that it would be really good to have some design sketches for frontend before implementing it. But I'm not a designer, so I'm open to suggestions. However, I can try to describe what I expect from the frontend:

1. Search bar that allows to search repositories by author/package name/etc.
2. Several filters near search bar that allow to filter by issue labels, package category, etc.
3. When search is performed, list of issues for all repositories satisfying selected filters should be shown.

It would be also really good to have some navigation bar with different elements, like what this project about, link to GitHub repo, search panel itself of course, login button for future improvements, and probably something else.

**Comment by Infi-Knight** [comment]
Thank you for the pointers. I will try to implement some mocks while I am drafting up the project proposal.

**Comment by Infi-Knight** [comment]
I have scaffolded a figma document to prototype the frontend at https://www.figma.com/file/gttFX0LgCaxDLnHwAzdHetJA/issue-wanted?node-id=0%3A1
